pgrx-sql-entity-graph/src/pgrx_sql.rs

```diff
@@ -390,13 +390,7 @@ impl PgrxSql {
             .neighbors_undirected(*item_index)
             .flat_map(|neighbor_index| match &self.graph[neighbor_index] {
                 SqlGraphEntity::Schema(s) => Some(String::from(s.name)),
-                SqlGraphEntity::ExtensionRoot(control) => {
-                    if !control.relocatable {
-                        control.schema.clone()
-                    } else {
-                        Some(String::from("@extname@"))
-                    }
-                }
```

Comment:
I don't understand the way Postgres handles schema-qualifying and substitutions very well, so I don't think I understand why this behavior was incorrect? Or becomes incorrect in the absence of the check, either way.

Reply:
`@extname@` propagates through and does not get substituted, to the point that `@extname@` finds its way verbatim into a Postgres expression and the parser errors with ``ERROR SQLSTATE[42601]: syntax error at or near "@"``.

Relocatable extensions, as their defining feature, can be moved to different schemas at runtime and as such you can't trust the control file to tell you what schema something is in. 

The reason we shouldn't be schema-qualifying based on the control file was @eeeebbbbrrrr 's decision but my understanding is that Postgres already does this more flexibly at runtime, and not making assumptions about this  would be more correct.

Reply:
...oh, amazing.